OvmfPkg/Library/GccLto/liblto-x64.s

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright (c) 2020, Intel Corporation. All rights reserved.
```

Comment:
Keep the original copyright info from Linaro as well because we copied from that file. Intel's should go below Linaro's.

Reply:
Rollback and added